component-samples/demo/scripts/fdorvconfig.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+#
+# Copyright 2024 Intel Corporation
+# SPDX-License-Identifier:
+#
+# This script configures FDO Components AIO/Owner to complete TO0 with Hosted RV.
+# Downloads the target hosted RV certificate and adds to ca-cert.pem of the mentioned components.
+# Disables AutoVoucherInjection worker and Enables StandardVoucherStorage worker in AIO, that enables TO0 trigger.
+# 
+# USAGE
+# bash fdorvconfig.sh -a <path_to_aio_directory> -o <path_to_owner_directory>
+# 
+# Example of providing custom proxy to use while connecting to target server
+# bash fdorvconfig.sh -a aio/ -o owner/ -p http://testproxy.com:4444
+# 
+# Example to configure either owner or AIO
+# bash fdorvconfig.sh -a aio/
+# bash fdorvconfig.sh -o owner/
+# 
+# NOTE
+# If any proxy is not provided by using argument '-p'
+# This script considers environment variables "https_proxy" and "HTTPS_PROXY" for connection to Hosted RV.
+# Default hosted instance is fdorv.com and port 443, but can be configured.
+
+
+
+############################################################
+# Help                                                     #
+############################################################
+Help()
+{
+    # Display Help
+    echo "This script is used to configure FDO AIO, Owner components to communicate with hosted RV instance fdorv.com (default)"
+    echo
+    echo "Syntax: bash fdorvconfig.sh [-a|h|o|p]"
+    echo "options:"
+    echo "a     Absolute or relative path to AIO directory"
+    echo "o     Absolute or relative path to Owner directory"
+    echo "p     Proxy to be used to connect to hosted RV, if not provided, uses environment variables https_proxy and HTTPS_PROXY"
+    echo "h     Help."
+    echo
+}
+
+while getopts a:ho:p: flag;
```

Comment:
is it a:h:o:p ?

Reply:
Since help `h` doesn't require any argument and proxy values `p` requires an argument, `:` is used for `p` and not `h`